dlrs/main/objects/DeclarativeLookupRollupSummaries__c/fields/DisableProblemEmails__c.field-meta.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@sharicarlson13 this is the Custom Setting field I'd like to add that disables the Problem Emails. Can you review the fullName, Label, helpText, and description?

Happy to change anything you recommend.

Reply:
The Field Name, Label, HelpText and Description look good to me. Thanks!